riemann/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM debian:8.1
```

Comment:
Does this have to be maintained here or can we reuse an existing reimann docker image?